src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -44,6 +44,17 @@ def _raise_error_if_not_drawing_classifier_input_sframe(
     if len(dataset) == 0:
         raise _ToolkitError("Input Dataset is empty!")
 
+def drop_missing_values(dataset, feature, is_train):
+    original_dataset_size = len(dataset)
+    dataset = dataset.dropna(columns=feature)
+    if is_train:
```

Comment:
Nit: this if/else should probably go under the if statement below, only if we're actually going to print something. Then it can also be a one liner:
```
if original_dataset_size != len(dataset):
  dataset_type = "train" if is_train else "validation"
  ...
```